src/libs/exception/exceptions.ts

```diff
@@ -0,0 +1,22 @@
+import { HttpException } from '@nestjs/common';
+import type { ErrorOption } from './index';
+
+export const badRequest = (message?: string, option?: ErrorOption) => {
+  message = message || 'Bad Request';
+  throw new HttpException({ message, errorMessage: option?.errorMessage }, 400);
+};
```

Comment:
https://docs.nestjs.com/exception-filters#built-in-http-exceptions

네스트에 내장된 에러 클래스를 사용안하고 커스텀한 이유가 그냥 간단히 사용하기 위해서 인가요?

Reply:
에러 코드 없이 걍 내장되어있는 클래스 사용해도 될 것 같은데여

<img width="772" alt="스크린샷 2022-11-21 오후 12 06 50" src="https://user-images.githubusercontent.com/63342911/202954782-12c023fd-d0a1-4d08-ae67-dcb5cf8bc8ca.png">

Reply:
클래스를 쓰게되면 new로 하는게 별로인것 같아서 그럼 해당 오류들을 한번씩 매핑하는건 어떨까요?

Reply:
new 로 하는 건 왜 별로에요? HttpException 랑 똑같은 거 아닌가요?

Reply:
```suggestion
export const badRequest = (message?: string, option?: ErrorOption) => {
  message = message || 'Bad Request';
  throw new BadRequestException({ message, errorMessage: option?.errorMessage });
};
```
윈디는 이렇게 수정하는걸 제안한거 아닌가요?

Reply:
수정했습니다